TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/TwoPaneLayout.kt

```diff
@@ -106,11 +107,10 @@ private sealed class Screen(val route: String) {
  */
 @Composable
 internal fun SinglePaneContainer(
+    navController: NavHostController,
```

Comment:
Why do we need to pass in the "navController" as a parameter? Is it because of the coming navigation refactor, or just to solve the current problem?

Reply:
Whoops good catch, that was left over from some navigation investigation, but it's not needed for the current fix - I'll take it out!